assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_grading_submission.vm

```diff
@@ -275,7 +275,7 @@
         #else
             #if($submissionType == 1 || $submissionType == 3)
                 #if (($submission.Submitted || !$alertGradeDraft) && $!value_feedback_text && !$!value_feedback_text.isEmpty())
-                    <p class="text-info">$tlang.getString("gradingsub.belisthe")</p>
+                    <p class="bg-info">$tlang.getString("gradingsub.belisthe")</p>
```

Comment:
`alert alert-info rounded p-3` looks even better

Reply:
Thanks @ottenhoff! Changed to p-2. p3 is too bigger

![image](https://github.com/user-attachments/assets/1bf4059b-69e6-4757-b0f9-e12107a688e4)

![image](https://github.com/user-attachments/assets/45d73aef-11fd-499e-84f9-9a2b2870ebbe)

![image](https://github.com/user-attachments/assets/537ea9b8-353b-4001-9fb2-fd7647780a5e)

Reply:
looks good